neuromation/api/plugins.py

```diff
@@ -0,0 +1,41 @@
+import numbers
+from typing import Any, Dict, Type, Union
+
+
+class ConfigConflict(Exception):
```

Comment:
What is the difference between already existing `ConfigError` and `ConfigConflict` exceptions?